setup.py

```diff
@@ -108,21 +108,19 @@ def _is_constant_id(name: str) -> bool:
 
 
 def _load_kzg_trusted_setups(preset_name):
-    """
-    [TODO] it's not the final mainnet trusted setup.
-    We will update it after the KZG ceremony.
-    """
-    file_path = str(Path(__file__).parent) + '/presets/' + preset_name + '/trusted_setups/testing_trusted_setups.json'
+    trusted_setups_file_path = str(Path(__file__).parent) + '/presets/' + preset_name + '/trusted_setups/trusted_setup_4096.json'
+    roots_of_unity_file_path = str(Path(__file__).parent) + '/presets/' + preset_name + '/trusted_setups/trusted_setup_4096_roots_of_unity.json'
 
-    with open(file_path, 'r') as f:
+    with open(trusted_setups_file_path, 'r') as f:
         json_data = json.load(f)
+        trusted_setup_G1_lagrange = json_data['g1_lagrange']
+        trusted_setup_G2 = json_data['g2_monomial']
```

Comment:
For consistency, I think we should rename this to `trusted_setup_G2_monomial`.

Reply:
Because these fields come from `/trusted_setups/trusted_setup_4096.json`, I don't think this is necessary.